technic/mod.conf

```diff
@@ -1,3 +1,3 @@
 name = technic
 depends = pipeworks, technic_worldgen, basic_materials
-optional_depends = mcl_core, mcl_sounds, default, bucket, mesecons, mesecons_mvps, digilines, digiline_remote, unified_inventory, dye, craftguide, i3, mtt, vizlib, moreores, mcl_buckets, mcl_explosions, mcl_craftguide, zzzz_exchangeclone_init
+optional_depends = mcl_core, mcl_sounds, default, bucket, mesecons, mesecons_mvps, digilines, digiline_remote, unified_inventory, dye, craftguide, i3, mtt, vizlib, moreores, mcl_raw_ores, mcl_buckets, mcl_explosions, mcl_craftguide, zzzz_exchangeclone_init
```

Comment:
This is not needed.
```suggestion
optional_depends = mcl_core, mcl_sounds, default, bucket, mesecons, mesecons_mvps, digilines, digiline_remote, unified_inventory, dye, craftguide, i3, mtt, vizlib, moreores, mcl_buckets, mcl_explosions, mcl_craftguide, zzzz_exchangeclone_init
```

Reply:
why not? don't we want `mcl_raw_ores` to be loaded before `technic` so it doesn't try to use a bunch of items that don't yet exist?

Reply:
Because they are only used in recipes, which don't require the items to exist.

Reply:
ok, i removed that commit